packages/toolpad-app/src/apiErrors.ts

```diff
@@ -0,0 +1,17 @@
+import { ValueOf } from './utils/types';
+
+export const API_ERROR_CODES = {
+  APP_NAME_EXISTS: 'APP_NAME_EXISTS',
```

Comment:
I'd consider namespacing them to distinguish them from errors created by node.js or other libraries
```suggestion
  APP_NAME_EXISTS: 'TOOLPAD_APP_NAME_EXISTS',
```